tests/src/rules/no-unresolved.js

```diff
@@ -26,10 +26,10 @@ function runResolverTests(resolver) {
       test({ code: 'import "./malformed.js"' }),
 
       rest({ code: 'import foo from "./bar";' }),
-      rest({ code: "import bar from './bar.js';" }),
-      rest({ code: "import {someThing} from './test-module';" }),
-      rest({ code: "import fs from 'fs';" }),
-      rest({ code: "import('fs');"
+      rest({ code: 'import bar from \'./bar.js\';' }),
```

Comment:
please revert unrelated formatting changes (if you're using prettier in your editor, you should disable it, and only run prettier via eslint autofix)